package.json

```diff
@@ -6,7 +6,7 @@
     "@arkweid/lefthook": "^0.7.7"
   },
   "scripts": {
-    "e2e": "docker rm cypress || echo \"\" && docker-compose --env-file .env -f docker-compose.yml -f docker-compose.cypress.yml up --remove-orphans --abort-on-container-exit --exit-code-from cypress && docker compose down && docker rm cypress",
+    "e2e": "docker rm cypress || echo \"\" && docker compose --env-file .env -f docker-compose.yml -f docker-compose.cypress.yml up --remove-orphans --abort-on-container-exit --exit-code-from cypress && docker compose down && docker rm cypress",
```

Comment:
is anyone using this? neither the new nor the old version build successfully for me

Reply:
I fixed the cmd while trying to run the e2e locally, but my efforts to run the e2e's were unsuccessful. but as far as the change go, I can go either way, it does need to be changed at some point since `docker-compose` has been removed and integrated into docker so its `docker compose` now